nuxeo-core/nuxeo-core/src/main/java/org/nuxeo/ecm/core/blob/ColdStorageHelper.java

```diff
@@ -71,6 +73,7 @@
 
     /**
      * Moves the main content associated with the document of the given {@link DocumentRef} to a cold storage.
+     * The permission {@value org.nuxeo.ecm.core.api.security.SecurityConstants#WRITE_COLD_STORAGE} is required.
```

Comment:
can you add `<p/>` before the new line `The permission {@value org.nuxeo.ecm.core.api.security.SecurityConstants#WRITE_COLD_STORAGE} is required.` and can you complete the `@throws NuxeoException`